components/BreadCrumb.js

```diff
@@ -0,0 +1,42 @@
+import Link from '@components/Link'
+import { ChevronRightIcon, HomeIcon } from '@heroicons/react/20/solid'
```

Comment:
We use react icons, you will be able to find the same icons there https://react-icons.github.io/react-icons/search?q=chevronright